src/test/compile-fail/try-block-maybe-bad-lifetime.rs

```diff
@@ -12,6 +12,9 @@
 
 #![feature(try_blocks)]
 
+#![inline(never)]
+fn do_something_with<T>(_x: T) {}
+
 // This test checks that borrows made and returned inside try blocks are properly constrained
```

Comment:
really we should make these UI tests, not compile-fail

Reply:
feel up for that? Also, I'd prefer a directory named something like `src/test/ui/rfc123-try-block`

Reply:
or just `try-block`